docs/dev-tools/chopsticks/overview.md

```diff
@@ -0,0 +1,412 @@
+To test XCM (Cross-Consensus Messaging) messages between networks, you can fork multiple parachains and a relay chain locally using Chopsticks. 
+
+|    Option    |      Description       |
+| :----------: | :--------------------: |
+| `relaychain` | Relaychain config file |
+| `parachain`  | Parachain config file  |
+
+
+For example, to fork Moonbeam, Astar, and Polkadot enabling XCM between them, you can use the following command:
+
+```bash
+npx @acala-network/chopsticks xcm \
+--r polkadot \
+--p moonbeam \
+--p astar
+```
+
+After running it, you should see output similar to the following:
+
+<div id="termynal" data-termynal>
+    <span data-ty="input"><span class="file-path"></span>npx @acala-network/chopsticks xcm \
+--r polkadot \
+--p moonbeam \
+--p astar</span>
+    <br>
+    <span data-ty>[13:46:07.901] INFO: Loading config file https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbeam.yml</span>
+    <span data-ty>    app: "chopsticks"</span>
+    <span data-ty>[13:46:12.631] INFO: Moonbeam RPC listening on port 8000</span>
+    <span data-ty>    app: "chopsticks"</span>
+    <span data-ty>[13:46:12.632] INFO: Loading config file https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/astar.yml</span>
+    <span data-ty>    app: "chopsticks"</span>
+    <span data-ty>        chopsticks::executor  TRACE: Calling Metadata_metadata</span>
+    <span data-ty>        chopsticks::executor  TRACE: Completed Metadata_metadata</span>
+    <span data-ty>[13:46:23.669] INFO: Astar RPC listening on port 8001</span>
+    <span data-ty>    app: "chopsticks"</span>
+    <span data-ty>[13:46:25.144] INFO (xcm): Connected parachains [2004,2006]</span>
+    <span data-ty>    app: "chopsticks"</span>
+    <span data-ty>[13:46:25.144] INFO: Loading config file https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/polkadot.yml</span>
+        <span data-ty>    app: "chopsticks"</span>
+    <span data-ty>        chopsticks::executor  TRACE: Calling Metadata_metadata</span>
+    <span data-ty>        chopsticks::executor  TRACE: Completed Metadata_metadata</span>
+    <span data-ty>[13:46:53.320] INFO: Polkadot RPC listening on port 8002</span>
+    <span data-ty>    app: "chopsticks"</span>
+    <span data-ty>[13:46:54.038] INFO (xcm): Connected relaychain 'Polkadot' with parachain 'Moonbeam'</span>
+    <span data-ty>    app: "chopsticks"</span>
+    <span data-ty>[13:46:55.028] INFO (xcm): Connected relaychain 'Polkadot' with parachain 'Astar'</span>
+    <span data-ty>    app: "chopsticks"</span>
+</div>
+
+Now you can interact with the forked chains using the ports specified in the output.
+
+## WebSocket Commands
+
+Chopstick's internal WebSocket server has special endpoints that allow the manipulation of the local Substrate chain.
+
+These are the methods that can be invoked and their parameters:
+
+???+ function "**dev_newBlock** (newBlockParams) — Generates one or more new blocks"
```

Comment:
Unsure about this way to add these commands. I belive @eshaben found  better way to document such things

Reply:
Inside of an expandable element is correct, but it is missing the styling that we typically use, so it looks a bit different, so I'll create a separate PR for that. For a page like this, I would stick with the tabs for Parameters and Example, but I think I would drop the Parameters table for a list in this case (with styled types). 

I'll send you an example